bundles/binding/org.openhab.binding.maxcube/src/main/java/org/openhab/binding/maxcube/internal/message/MessageType.java

```diff
@@ -15,5 +15,19 @@
 * @since 1.4.0
 */
 public enum MessageType {
-	H, M, C, L, S
+	H("H:"),
+	M("M:"),
+	C("C:"), 
+	L("L:"),
+	S("S:");
+	
+	private String messageIndicator;
```

Comment:
this message indicatior is more or less duplicate of the message.getType() can you see if you can use that instead?